api/v2/views/base.py

```diff
@@ -24,8 +24,9 @@ def unresolved_requests_only(fn):
     @wraps(fn)
     def wrapper(self, request, *args, **kwargs):
         instance = self.get_object()
+        user_can_act = request.user.is_staff or request.user.is_superuser or CloudAdministrator.objects.filter(user=request.user.id)
```

Comment:
that examples seems add to me: 
```python
entry = Entry.objects.get(pk=123)
if some_queryset.filter(pk=entry.pk).exists():
    print("Entry contained in queryset")
```

Couldn't you just do: 
```python 
if some_queryset.filter(pk=123).exists():
    entry = Entry.objects.get(pk=123)
    # do work on `entry`
    print("Entry contained in queryset")
```

Maybe it's just the scenario in the example lacks _full_ context.